### DISCUSSION TITLE
Third mobile control option or rework of existing

### DISCUSSION THREAD

**Discussion by TheOnlyJoeEnderman** [agateau/pixelwheels#319] [Ideas]
The existing control schemes work, but I am not a fan of either. The pie puts the right steering too low, and the side buttons removes the reverse button. You can hold both to reverse, but it isn't great. Otherwise, I must say, it's a fun little game very reminiscent of old arcade racing games.

**Comment by agateau** [comment]
I can look into this. Do you have any suggestions on control schemes you like in other games?

**Comment by TheOnlyJoeEnderman** [comment]
If the pie had a variant with the controls horizontal instead of diagonal, that would help. Or, you could split the screen into four zones with the top two being left and right and the bottom two being reverse and use item. I'm not sure, but those might be good. Hopefully this at least inspires you some.

**Comment by agateau** [comment]
Sorry for the late reply and thanks for the suggestion, will keep it in mind.

**Reply by TheOnlyJoeEnderman** [reply]
No problem, thanks for making a great ad-free game.
